qiskit_nature/circuit/library/ansaetze/__init__.py

```diff
@@ -0,0 +1,23 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Ansatz circuits and utilities thereof."""
```

Comment:
I am not sure about `/library/ansaetze/` - we used to use the `ansatzes` as the plural - its listed as the plural of ansatze and is more easily recognizable I think to English speakers,

Reply:
What do you mean by "its listed as the plural of ansatze"?
I don't think the recognition factor is really decreased here..

Reply:
> What do you mean by "its listed as the plural of ansatze"?

Sorry I meant if go look up ansatze then it will 'list' ansatzes as the plural of that word.